bedevere/backport.py

```diff
@@ -37,98 +36,42 @@ async def _remove_backport_label(gh, original_issue, branch, backport_pr_number)
     Also leave a comment on the original PR referencing the backport PR.
     """
     backport_label = BACKPORT_LABEL.format(branch=branch)
-    if backport_label not in util.labels(original_issue):
+    if backport_label not in util.labels_(original_issue):
         return
     await gh.delete(original_issue['labels_url'], {'name': backport_label})
     message = MESSAGE_TEMPLATE.format(branch=branch, pr=backport_pr_number)
     await gh.post(original_issue['comments_url'], data={'body': message})
 
-
-@router.register("pull_request", action="opened")
-@router.register("pull_request", action="edited")
-async def manage_labels(event, gh, *args, **kwargs):
-    if event.data["action"] == "edited" and "title" not in event.data["changes"]:
-        return
-    pull_request = event.data["pull_request"]
-    title = util.normalize_title(pull_request['title'],
+async def manage_labels(gh, *args, **kwargs):
+    with open(os.environ["GITHUB_EVENT_PATH"]) as f:
```

Comment:
Curious about retrieving the `GITHUB_EVENT_PATH from the environment variable. Is this because you're testing/debugging locally?

Reply:
Oh, not really, it's to use data for GitHub with the action triggered instead of making an API call to GitHub API to get that. 
I can get data like type of the event, name of the PR and so on. With that I only make request of something I don't have with the event.